includes/class-sensei-learner.php

```diff
@@ -685,4 +709,41 @@ public static function find_by_query_var( $query_var ) {
 
 		return get_user_by( 'login', $query_var );
 	}
+	/**
+	 * Returns course list for AJAX "more" call on Students page.
+	 *
+	 * @return void
+	 */
+	public function get_course_list() {
+		if ( empty( $_POST['nonce'] ) ) {
+			wp_send_json_error( array( 'error' => esc_html__( 'Insufficient Permissions.', 'sensei-lms' ) ) );
+		}
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Don't modify the nonce.
+		if ( ! isset( $_POST['user_id'] ) || ! wp_verify_nonce( ( wp_unslash( $_POST['nonce'] ) ), 'get_course_list' ) ) {
```

Comment:
Extraneous  parentheses here. 

```suggestion
		if ( ! isset( $_POST['user_id'] ) || ! wp_verify_nonce( wp_unslash( $_POST['nonce'] ), 'get_course_list' ) ) {
```